dashboard/app/controllers/datasets_controller.rb

```diff
@@ -5,8 +5,13 @@ class DatasetsController < ApplicationController
   before_action :initialize_firebase
   authorize_resource class: false
 
+  LIVE_DATASETS = ['Daily Weather', 'Top 200 USA', 'Top 200 Worldwide', 'Viral 50 USA', 'Viral 50 Worldwide']
```

Comment:
Is there a way to make this localized?

Reply:
What do you mean localized? These are the names of the live tables, which is not translated/translatable.

Reply:
Not blocking: Is translating these titles something we'd ever need to worry about? Or if we wanted to internationalize this feature would we instead import data tables with different languages?